### ISSUE TITLE
HueSaturationValue for float32 dtype [0..1].

### ISSUE THREAD

**Issue by thingumajig** [albumentations-team/albumentations#690]
HueSaturationValue problem for float32 [0..1] images:

Original augmentation list:
```
transforms = a.Compose([
    a.RandomRotate90(),
    a.Flip(),
    a.OneOf([
        a.MotionBlur(p=0.2),
        a.MedianBlur(blur_limit=3, p=0.1),
        a.Blur(blur_limit=3, p=0.1),
    ], p=0.3),

    a.ShiftScaleRotate(shift_limit=0.0625, scale_limit=0.1, rotate_limit=45, p=0.5),
    a.OneOf([
        a.OpticalDistortion(p=0.3),
        a.GridDistortion(num_steps=10, p=0.3),
        a.ElasticTransform(p=0.4),
    ], p=0.6),        
    
    a.HueSaturationValue(hue_shift_limit=0.1, sat_shift_limit=0.2, val_shift_limit=0.1, p=1.0),

     a.GaussNoise(var_limit=(0.002, 0.006), p=0.4),   
     a.RandomBrightnessContrast(p=0.3),
])

```
Visualization code:
```
print(f'Albumentation version: {a.__version__}')

image, label = next(iter(train_ds)) # extract 1 batch from the dataset

fig = plt.figure(figsize=(30, 30))
def vis_tr(img, row, col, title, cols=5):
    print(f'{title}:\t{type(img)}-{img.dtype}\tMin: {img.min()},\tMax: {img.max()}\tMean: {img.mean()}')

    img = img * 255.
    img = img.astype("uint8")
    ax = fig.add_subplot(4, 5, row*cols+col, xticks=[], yticks=[])
    ax.imshow(img)
    ax.set_title(title)
    
for i in range(4):
    img = image[i]
    p_img = img.numpy()
    p_img = p_img / 255.
    vis_tr(p_img, i, 1, "original image")
    
    vis_tr(transforms(image=p_img)['image'], i, 2, "augmented 1")
    vis_tr(transforms(image=p_img)['image'], i, 3, "augmented 2")
    vis_tr(transforms(image=p_img)['image'], i, 4, "augmented 3")
    vis_tr(transforms(image=p_img)['image'], i, 5, "augmented 4")
```
### When I test augmentation, I get images whose hue is shifted towards the green(for all generated images):
![image](https://user-images.githubusercontent.com/8703531/91673827-a5c6f280-eb3e-11ea-8200-98130000a200.png)

## My workaround

```
transforms = a.Compose([
    a.RandomRotate90(),
    a.Flip(),

    a.OneOf([
        a.MotionBlur(p=0.2),
        a.MedianBlur(blur_limit=3, p=0.1),
        a.Blur(blur_limit=3, p=0.1),
    ], p=0.3),

    a.ShiftScaleRotate(shift_limit=0.0625, scale_limit=0.1, rotate_limit=45, p=0.5),
    a.OneOf([
        a.OpticalDistortion(p=0.3),
        a.GridDistortion(num_steps=10, p=0.3),
        a.ElasticTransform(p=0.4),
    ], p=0.6),        
    
    # Changed block
    a.FromFloat(dtype='uint8', max_value=255., always_apply=True),
    a.HueSaturationValue(hue_shift_limit=10, sat_shift_limit=10, val_shift_limit=10, p=1.0),
#     a.HueSaturationValue(hue_shift_limit=0.1, sat_shift_limit=0.2, val_shift_limit=0.1, p=1.0),
    a.ToFloat(max_value=255.0, always_apply=True),


     a.GaussNoise(var_limit=(0.002, 0.006), p=0.4),   
     a.RandomBrightnessContrast(p=0.3),
])

```
### Correct output (without green):
![image](https://user-images.githubusercontent.com/8703531/91674022-a2803680-eb3f-11ea-9e14-c2648425f740.png)

## Environment

 - Albumentations version : 0.4.6
 - Python version: 3.6.9
 - OS :  Ubuntu 18.04.5 LTS (GNU/Linux 4.15.0-112-generic x86_64)
 - How you installed albumentations: pip, virtualenv
 

### Thanks for the handy tool 👍

**Comment by Dipet** [comment]
Thank you for information. You must be careful with float dtype, because in float hue is measured in degrees in range `[0, 360]`

**Comment by Dipet** [comment]
Was fixed in #696 , try to install latest version from github.
